### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by jodyngo** [wenguanwang/ASNet#11]
May you please provide the training file with the proposed loss function as described in the paper?
